src/index.html

```diff
@@ -4,9 +4,516 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="catalog">
+    <header class="header catalog__header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="Moyo_logo">
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+              id="apple"
+            >
+              Apple
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a
+              href="#samsung" class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a
+              href="#phones" class="nav__link"
+            >
+              Смартфоны и телефоны
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              Гаджеты
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a
+              href="#pads"
+              class="nav__link"
+            >
+              Планшеты
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              Фото
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog__container">
```

Comment:
In in body we would have `page` class, then there 
```suggestion
    <main class="catalog page__catalog">
```

Reply:
Thanks!!